.github/workflows/ci.yml

```diff
@@ -57,6 +57,8 @@ jobs:
     runs-on: windows-latest
     steps:
       - name: Activate msys2 & mingw64
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Ideally the `add-path` commands need to be fixed, not just bypassed. At first glance an environment file should work for what we're doing here.

https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Reply:
@embeddedt see `ci*` branches in https://github.com/puzrin/dispenser. I failed to fix that in better way. If anyone will be able to do that - cool (but not mandatory)